emqx.conf

```
@@ -30,4 +30,15 @@ retainer {
     backend {
         storage_type = disc
     }
-}
\ No newline at end of file
+}
+plugins {
+    states = [
+        {
+         
+            name_vsn = "mqtt2pgsql-1.0.0",
+
+            enable = true
```

@Mohan15054  Pleaes remveo space lines 


```suggestion

plugins {
    states = [
        {
            name_vsn = "mqtt2pgsql-1.0.0",
            enable = true
        }
    ]
    install_dir = "plugins"
}
```